server.js

```diff
@@ -14,7 +14,10 @@ require('dotenv').config({ path: '.variables.env' });
 // Connect to our Database and handle any bad connections
 // mongoose.connect(process.env.DATABASE);
 
-mongoose.connect(process.env.DATABASE);
+mongoose
+  .connect(process.env.DATABASE)
+  .then(() => console.log('Database has been connected successfully'));
```

Comment:
It's better if you check if this is on non-production environment to console log this text

Reply:
Could you please elaborate what are the problems if this text is logged in a production environment, I am just asking out of curiosity. = )

Reply:
All console logs/errors must not be logged on production as a general rule, first of all, it's noise on the devtools console especially such logs, there are some logs are may expose some platform content, typically console logs/infos are for development only, console warnings/errors must be used with a logger/monitoring service instead and keep devtools console only for uncaught messages

Reply:
You are right!